src/test/kotlin/org/malachite/estella/BaseIntegration.kt

```diff
@@ -62,7 +62,7 @@ class BaseIntegration {
         path: String,
         method: HttpMethod,
         headers: Map<String, String> = mapOf(),
-        body: Map<String, Any> = mapOf()
+        body: Any = mapOf<String, Any>()
```

Comment:
Czemu Any?

Reply:
No w body może to być problemem. O ile w paramie w ścieżce to jest easy (podajesz po prostu w ścieżce), o tyle nie bardzo nawet wiem, jakby to miało wtedy wyglądać.